src/sentry/tasks/post_process.py

```diff
@@ -178,68 +178,81 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
 
         _capture_stats(event, is_new)
 
-        # we process snoozes before rules as it might create a regression
-        has_reappeared = process_snoozes(event.group)
-
-        handle_owner_assignment(event.project, event.group, event)
-
-        rp = RuleProcessor(event, is_new, is_regression, is_new_group_environment, has_reappeared)
-        has_alert = False
-        # TODO(dcramer): ideally this would fanout, but serializing giant
-        # objects back and forth isn't super efficient
-        for callback, futures in rp.apply():
-            has_alert = True
-            safe_execute(callback, event, futures)
-
-        if features.has(
-            'projects:servicehooks',
-            project=event.project,
-        ):
-            allowed_events = set(['event.created'])
-            if has_alert:
-                allowed_events.add('event.alert')
-
-            if allowed_events:
-                for servicehook_id, events in _get_service_hooks(project_id=event.project_id):
-                    if any(e in allowed_events for e in events):
-                        process_service_hook.delay(
-                            servicehook_id=servicehook_id,
-                            event=event,
-                        )
-
-        if event.get_event_type() == 'error' and _should_send_error_created_hooks(event.project):
-            process_resource_change_bound.delay(
-                action='created',
-                sender='Error',
-                instance_id=event.event_id,
-                project_id=event.project_id,
-                group_id=event.group_id,
-            )
-        if is_new:
-            process_resource_change_bound.delay(
-                action='created',
-                sender='Group',
-                instance_id=event.group_id,
-            )
-
-        for plugin in plugins.for_project(event.project):
-            plugin_post_process_group(
-                plugin_slug=plugin.slug,
+        if event.group:
+            process_group(
                 event=event,
                 is_new=is_new,
-                is_regresion=is_regression,
+                is_regression=is_regression,
                 is_sample=is_sample,
+                is_new_group_environment=is_new_group_environment,
             )
 
         event_processed.send_robust(
```

Comment:
This is the only statement we need to run out of post_processing for issueless events.
When/if this becomes a performance problem. This can be moved earlier (directly into eventstream) and skip the post process task altogether. That will be a simple change